nni/algorithms/compression/pytorch/quantization/quantizers.py

```diff
@@ -145,8 +145,11 @@ def __init__(self, model, config_list, optimizer=None):
                     state where activation quantization ranges do not exclude a signiﬁcant fraction of values, default value is 0
                 - op_types : list of string
                     types of nn.module you want to apply quantization, eg. 'Conv2d'
+                - dummy_input : tuple of tensor
+                    inputs to the model, which are used to get the graph of the module
```

Comment:
I think we also need to explain the graph is used to find bn after Conv. If user doesn't provide `dummy_input`, the quantizer will not fold bn.

Reply:
done